database/migrations/2023_08_23_045341_alter_post_fk_column_comments_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('comments', function(Blueprint $table){
+            $table->dropForeign('comments_user_id_foreign');
+            $table->dropForeign('comments_post_id_foreign');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        //
```

Comment:
Must implement down function for changing table definition. 
It must be reverted for modified code. 
Eg. If you drop foreign key in up(), then you must add that foreign key in down().  If you rename table column in up(), then you must reset previous column name in down().